website/docs/tutorials/06-deploying/02-migrations/invoke.md

```diff
@@ -19,7 +19,6 @@ It's an extension of [CheatcodeNetworkConfig](../03-network-config.md), so it's
 Wei = int
 class SignedCheatcodeConfig(CheatcodeNetworkConfig):
     max_fee: Wei | None
```

Comment:
```suggestion
    max_fee: Fee
```

Reply:
[change type of max_fee to Fee](https://github.com/software-mansion/protostar/pull/979/commits/8b3d0ca5d7f7c558635efaccc2bc6c78e0f11c82)